lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr.c

```diff
@@ -486,21 +488,25 @@ status_code_e esm_ebr_set_status(
   OAILOG_FUNC_IN(LOG_NAS_ESM);
 
   if (emm_context == NULL) {
-    OAILOG_ERROR(LOG_NAS_ESM, "ESM-FSM   - emm context null... \n");
+    OAILOG_ERROR_UE(
+        LOG_NAS_ESM, emm_context->_imsi64,
+        "ESM-FSM   - emm context null... \n");
     OAILOG_FUNC_RETURN(LOG_NAS_ESM, RETURNerror);
   }
 
   if ((ebi < ESM_EBI_MIN) || (ebi > ESM_EBI_MAX)) {
-    OAILOG_ERROR(
-        LOG_NAS_ESM,
+    OAILOG_ERROR_UE(
+        LOG_NAS_ESM, emm_context->_imsi64,
         "ESM-FSM   - Invalid EPS bearer identity range ebi= (%d) \n", ebi);
     OAILOG_FUNC_RETURN(LOG_NAS_ESM, RETURNerror);
   }
 
   ue_mm_context_t* ue_mm_context =
       PARENT_STRUCT(emm_context, struct ue_mm_context_s, emm_context);
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(LOG_NAS_ESM, "ESM-FSM   - ue mme context null... \n");
+    OAILOG_ERROR_UE(
```

Comment:
ue_mm_context does not exist in this case, let's not change this

Reply:
Resolved